src/style.css

```diff
@@ -1,3 +1,84 @@
+:root {
+  --header-bg: #fff;
+  --header-height: 60px;
+  --nav-link-color: #000;
+  --nav-link-hover-color: #00acdc;
+  --nav-font-size: 12px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--header-bg);
+  width: 100%;
+  padding: 0 50px;
+}
+
+.logo_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+}
+
+.nav_list .nav_item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--nav-link-color);
+  font-family: Roboto, sans-serif;
+  font-size: var(--nav-font-size);
+  font-weight: 500;
+  height: var(--header-height);
+  line-height: 14.6px;
+}
+
+.nav_link:hover {
+  color: var(--nav-link-hover-color);
+}
+
+.is-active {
+  color: var(--nav-link-hover-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background: var(--nav-link-hover-color);
+  position: absolute;
+  border-radius: 8px;
+  bottom: 0;
+}
```

Comment:
The `is-active::after` pseudo-element is used to underline the active link. Ensure that this styling is visible and effectively indicates the active state, especially if the background or other styles change.